ds_judgements_public_ui/sass/includes/_document_text.scss

```diff
@@ -1,4 +1,4 @@
-.judgment {
```

Comment:
Just wanted to check - do we need to make a corresponding change in EUI to make sure that this CSS gets pulled across correctly on build?

Reply:
(see the note in the readme)

Reply:
https://github.com/nationalarchives/ds-caselaw-editor-ui/pull/1073